code/__DEFINES/_globals.dm

```diff
@@ -60,3 +60,6 @@
 
 /// Create a typed null global
 #define GLOBAL_DATUM(X, Typepath) GLOBAL_RAW(Typepath/##X); GLOBAL_UNMANAGED(X)
+
+/// Used to determine BYOND static init order
+#define STATIC_INIT(expression) world._(##expression)
```

Comment:
do we need this if profiling tools aren't in use?

Reply:
Depends on how much control we want over init order.

With it as it is we can (%99.99) guarantee Master and GLOB will be initialized before statics run.